dashboard/app/views/shared/_school_info_registration.html.haml

```diff
@@ -0,0 +1,101 @@
+- require 'country_codes'
```

Comment:
I considered this, but decided to take this on in the next iteration (which will be very soon, we're going to add a school dropdown interstitial for teachers that don't fill this info out) because the discussions around that are veering towards substantial UI and functionality changes that are likely to be applied in all the places where this shows up.

I definitely agree that this should happen, though.